src/main/kotlin/org/rust/openapiext/utils.kt

```diff
@@ -125,3 +128,7 @@ inline fun <T> UserDataHolderEx.getOrPutSoft(key: Key<SoftReference<T>>, default
         val value = defaultValue()
         putUserDataIfAbsent(key, SoftReference(value)).get() ?: value
     }
+
+const val PLUGIN_ID: String = "org.rust.lang"
+
+val PLUGIN: IdeaPluginDescriptor get() = PluginManager.getPlugin(PluginId.getId(PLUGIN_ID))!!
```

Comment:
It still looks like a constant
I'd prefer something like `fun plugin(): IdeaPluginDescriptor = PluginManager.getPlugin(PluginId.getId(PLUGIN_ID))!!`